packages/components/src/theme/ComposerTheme/Components/Input.ts

```diff
@@ -0,0 +1,361 @@
+        color: 'text-secondary',
+        textStyle: 'text-body-regular',
+      },
+      _hover: {
+        backgroundColor: 'background-action-hover',
+        border: 'solid 0.063rem var(--chakra-colors-ui-element-outline-active)',
+      },
+      // We need to remove the focus state from the chakra input so we can add the border shadow to the full input group.
+      _focus: {
+        boxShadow: 'none',
+        border:
+          'solid 0.063rem var(--chakra-colors-ui-element-outline-default)',
+        borderColor: 'ui-element-outline-default',
+      },
+      _invalid: {
+        boxShadow: 'none',
+        border: 'none',
+        borderColor: 'transparent',
+      },
+      _disabled: {
+        _placeholder: {
+          color: 'text-disabled',
+        },
+        color: 'text-secondary',
+        backgroundColor: 'background-action-disabled',
+        border:
+          'solid 0.063rem var(--chakra-colors-ui-element-outline-disabled)',
+      },
+    },
+  };
+};
+
+export const flushedStyle = (props) => {
+  const inputGroupConditionalStyle = getInputGroupStyle(props, 'flushed');
+  const labelConditionalStyle = getLabelStyle(props, 'flushed');
+  const iconConditionalStyle = getIconStyle(props);
+
+  return {
+    inputGroup: {
+      borderRadius: 0,
+      ...inputGroupConditionalStyle,
+    },
+    leftLabel: {
+      ...labelStyles,
+      ...labelConditionalStyle,
+    },
+    rightLabel: {
+      ...labelStyles,
+      ...labelConditionalStyle,
+    },
+    element: {
+      color: 'text-secondary',
+      ...iconConditionalStyle,
+    },
+    elementContainer: {
+      height: '100%',
+    },
+    field: {
+      color: 'text-primary',
+      backgroundColor: 'background-action-default',
+      border: 'none',
+      borderColor: 'ui-element-outline-default',
+      borderRadius: 0,
+      _placeholder: {
+        color: 'text-secondary',
+        textStyle: 'text-body-regular',
+      },
+      _hover: {
+        backgroundColor: 'background-action-hover',
+        border: 'none',
+        borderBottom:
+          '0.063rem solid var(--chakra-colors-ui-element-outline-active)',
+      },
+      // We need to remove the focus state from the chakra input so we can add the border shadow to the full input group.
+      _focus: {
+        boxShadow: 'none',
+        border: 'none',
+        borderBottom:
+          'solid 0.063rem var(--chakra-colors-ui-element-outline-default)',
+        borderColor: 'ui-element-outline-default',
+      },
+      _invalid: {
+        boxShadow: 'none',
+        border: 'none',
+        borderColor: 'transparent',
+      },
+      _disabled: {
+        _placeholder: {
+          color: 'text-disabled',
+        },
+        color: 'text-secondary',
+        backgroundColor: 'background-action-disabled',
+        border:
+          'solid 0.063rem var(--chakra-colors-ui-element-outline-disabled)',
+      },
+    },
+  };
+};
+
+export const Input: ComponentStyleConfig = {
+  baseStyle,
```

Comment:
do we need this or should we use a default variant instead?

Reply:
added default variant instead of using base style